### ISSUE TITLE
Auto-scaling results in nodes reaching 100% JVM heap memory resource and becoming unresponsive

### ISSUE THREAD

**Issue by timantmedia** [ant-media/Ant-Media-Server#5446]
### Short description

The  heap memory always gets to 100% utilisation and then crashes the instance which becomes inaccessible:

#### What version of Ant Media Server are you using?
Enterprise Edition 2.6.0 20230517_0442

#### How many broadcasts/viewers are you handling with your server?

2 and 6000 viewer

#### What are your server specifications?

c5.9xlarge

#### How are you publishing live streams e.g RTMP, SRT, WebRTC and what encoder are you using?

RTC

### Environment

* Operating system and version:
* Java version: 
* Ant Media Server version: 2.6.0
* Browser name and version:

### Steps to reproduce
1. 
2. 
3.

### Expected behavior
__Put as much detail here as possible__

### Actual behavior
__Put as much detail here as possible__

 

### Logs
__Place logs on [pastebin](http://pastebin.com/) or elsewhere and put links here__

Ask your questions on [Ant Media Github Discussions](https://github.com/orgs/ant-media/discussions)

**Comment by mekya** [comment]
Hi @timantmedia,

I think it's an expected behaviour to encounter problem in c5.9xlarge with 6000 viewer. 

We provide solution to the user with correct deployment options. I've put this to the Next Sprint and increased the priority

**Comment by timantmedia** [comment]
@mekya @muratugureminoglu Could you confirm what's happening with this? Is the other JVM heap memory issue being investigated as the same issue? Where are the "We provide a solution to the user with correct deployment options"?

**Comment by timantmedia** [comment]
@muratugureminoglu @mekya is there any update on this issue, I am not sure what was the outcome.

**Comment by mekya** [comment]
This is already in Icebox stage. It means that it's not supposed to have updates.  I think there is some confusion in the issues you follow up.

**Comment by timantmedia** [comment]
@mekya apologies, I think I am confused with what the updates were on this issue. The previous response is "We provide solution to the user with correct deployment options. I've put this to the Next Sprint and increased the priority" does this mean we already resolved this issue then?

**Comment by mekya** [comment]
> @mekya apologies, I think I am confused with what the updates were on this issue. The previous response is "We provide solution to the user with correct deployment options. I've put this to the Next Sprint and increased the priority" does this mean we already resolved this issue then?

No. After the message above, I've set status to Icebox. As far as I remember, the issue is not reproducible in our test with the user. 

Additionally, we've provided a solution to other memory leakages issues for the users. 

So I think we can keep this in Icebox or even close it.
